### PR TITLE
Media gallery import

### PR DESCRIPTION
**Pull request by tmotyl**
needs to be tested

### CODE REVIEW

**Comment by shyang292** [comment]
doesn't work for importing media gallery

File: magmi/plugins/base/itemprocessors/configurables/magmi_configurableprocessor.php [hunk 1]
```diff
@@ -152,6 +161,11 @@ public function getMatchMode($item)
 
     public function processItemAfterId(&$item, $params = null)
     {
+        if (!isset($item['type']) || $this->_disabled == true) {
+            //不处理
```
Comment (tmotyl) [inline magmi/plugins/base/itemprocessors/configurables/magmi_configurableprocessor.php]: please make comments only in English

File: magmi/plugins/extra/itemprocessors/imageprocessor/imageitattributeemprocessor.php [hunk 2]
```diff
@@ -255,10 +255,10 @@ public function getImageId($pid, $attid, $imgname, $refid = null, $store_id = 0)
 				(?,?)";
             $imgid = $this->insert($sql, array($attid, $imgname));
         } else {
-            $sql = "UPDATE $t
-				 SET value=?
-				 WHERE value_id=?";
-            $this->update($sql, array($imgname, $imgid));
+            //$sql = "UPDATE $t
```
Comment (tmotyl) [inline magmi/plugins/extra/itemprocessors/imageprocessor/imageitattributeemprocessor.php]: why is this commented? If it's not needed please explain why and remove it

File: magmi/web/security.php [hunk 3]
```diff
@@ -1,5 +1,6 @@
 <?php
 require_once(dirname(dirname(__FILE__))."/inc/magmi_auth.php");
+return;
```
Comment (tmotyl) [inline magmi/web/security.php]: this makes magmi insecure, probably was committed by mistake.

**Comment by tmotyl** [comment]
fixed with https://github.com/macopedia/magmi-git/pull/13

**Pull request closed without merging**